app/views/posts/index.html.erb

```diff
@@ -23,21 +22,15 @@
     <div>
       <h3>Comments</h3>
       <% post.comments.each do |comment| %>
-        <p>
-          This will be updated to name:
-          <%= comment.body %>
-        </p>
+        <p> <%= comment.user.name %> said: <%= comment.body %></p>
       <% end %>
     </div>
     <div>
       Add a comment:
       <%= form_for([post, post.comments.build]) do |comment| %>
-        <p>
-          <%= comment.text_area :body %>
-        </p>
-        <p>
-          <%= comment.submit %>
-        </p>
+      <%= comment.hidden_field(:user_id, value: current_user.id) %>
```

Comment:
Wouldn't it be nice if we didn't have to keep playing this expensive game where I turn into a human indentation checker!